specs/core/0_beacon-chain.md

```diff
@@ -670,8 +670,8 @@ full_deposit_count: uint256
 @payable
 @public
 def deposit(deposit_input: bytes[2048]):
-    assert msg.value >= as_wei_value(MIN_DEPOSIT_AMOUNT, "gwei")
-    assert msg.value <= as_wei_value(MAX_DEPOSIT_AMOUNT, "gwei")
+    assert msg.value >= MIN_DEPOSIT
```

Comment:
In line with using Gwei everywhere, my suggestion is to define `deposit_amount` (e.g. `deposit_amount = msg.value / WEI_PER_GWEI`) and then only deal with Gwei amounts (never `msg.value` directly).